src/Components/Pickers/Internal/Picker.base.styles.ts

```diff
@@ -6,24 +6,36 @@ const classNames = {
     button: `${classPrefix}-button`
 };
 export const getStyles = (_props: IPickerBaseStyleProps): IPickerBaseStyles => {
+    const { theme } = _props;
```

Comment:
nit: _props can be renamed to props if it is being used. The underscore is used to avoid a lint warning for unused arguments.